EAXARG-456d27 fill dest=0x939b80

(rationale: see preceding patch in the series)

--- a/dd2.c
+++ b/dd2.c
@@ -35081,7 +35081,7 @@
     Debug_Stub();
     FUN_00420c4c();
     FUN_00422da4();
-    FUN_00456d27(2,0);
+    (_g_eax=0x939b80, FUN_00456d27(2,0));
     if ((_DAT_0075444a & 0x80) == 0) {
       if ((_DAT_0075444a & 0x20) != 0) {
         Play_Click_FX();
